src/components/navigation/accountLinks.js

```diff
@@ -78,11 +78,21 @@ const AccountLinks = ({ mainLinkClasses }) => {
                   className={`su-inline-block su-mr-10 ${
                     showDesktop ? '' : 'su-sr-only'
                   }`}
-                >{`Hi, ${userProfile.name.fullNameParsed.firstName} ${userProfile.name.fullNameParsed.lastName}`}</span>
+                >
+                  Hi,
+                  {userProfile?.name?.fullNameParsed
```

Comment:
```suggestion
                  {userProfile?.name?.digitalName
```

Shouldn't this be digital name first AND then fall back to first/last when not available?